File: ai/app/schemas/train_response.py
from pydantic import BaseModel

class TrainResponse(BaseModel):
    modelKey:str
    precision: float
    recall: float
    mAP50: float
    mAP5095: float
    fitness: float

class ClassificationTrainResponse(BaseModel):
    modelKey: str
    accuracy: float
    fitness: float

File: backend/src/main/java/com/worlabel/domain/model/entity/dto/TrainResponse.java
package com.worlabel.domain.model.entity.dto;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.ToString;

@Getter
@ToString
@AllArgsConstructor(access = AccessLevel.PRIVATE)
public class TrainResponse {

    private String modelKey;

    private double precision;

    private double recall;

    private double mAP50;

    private double mAP5095;

    private double fitness;

    private double accuracy;
}
